developing-packages.md

```diff
@@ -162,7 +162,37 @@ You can run the example app by pressing the &#9654; button.
 
 Finally, you need to connect the API written in Dart code with the platform-specific
 implementations. This is done using [platform channels](/platform-channels/).
+
+## Adding documentation
+
+It is recommended practice to add the following documentation to all packages:
+
+1. A `readme.md` file that introduces the package
+1. A `changelog.md` file that documents changes in each version
```

Comment:
should this be README.md and CHANGELOG.md?

Also, do we really want to say you need a change log?